apps/shipments/models.py

```diff
@@ -373,7 +375,7 @@ def set_vault_uri(self, vault_uri):
                                                    'module': __name__})
         return async_job
 
-    def set_vault_hash(self, vault_hash, action_type, rate_limit=settings.DATA_VAULT_HASH_RATE_LIMIT,
+    def set_vault_hash(self, vault_hash, action_type, rate_limit=-1,
```

Comment:
```suggestion
    def set_vault_hash(self, vault_hash, action_type, rate_limit=self.manual_update_hash_interval,
```

Reply:
The reason this change was implemented was because I am unable to include self.manual_update_hash_interval as it is included in the parameters of the call. I could set it settings.DATA_VAULT_HASH_RATE_LIMIT and if rate_limit just use self.manual_update_hash_interval. Wasn't sure about the best implementation

Reply:
Unfortunately your workaround fails if a custom rate_limit is passed into set_vault_hash.

set_vault_hash should support:

- rate_limit parameter is omitted, which should use the manual hash rate limit
- a custom rate_limit is passed to the function, which should use that rate limit
- rate_limit is explicitly set to None, which should perform an _instant_ hash update

I _really_ don't like using the value of -1 to represent the first scenario. I think `None` is most appropriate but that clashes with the third scenario. Which means we must tweak that requirement to be:
- rate_limit is explicitly set to 0, which should perform an _instant_ hash update

Which will still work with the `elif rate_limit:` logic below.

And actually, I see we are using the value 0 anyways in apps/shipments/events.py, so there is no reason why we shouldn't do something like 
```
set_vault_hash(rate_limit=None){
  if rate_limit is None:
    rate_limit = self.manual_update_hash_interval
}
```